digital_business_card/src/index.js

```diff
@@ -0,0 +1,6 @@
+import React from "react"
+import ReactDOM from "react-dom/client"
+import DigitalCard from "./DigitalCard"
+
+// ReactDOM.render(<DigitalCard />, document.getElementById("root"))
```

Comment:
why do we need this line?
Add a comment explaining what React version do you need?